include/abb_librws/rws_interface.h

```diff
@@ -419,12 +419,19 @@ class RWSInterface
    */
   StaticInfo collectStaticInfo();
 
+  /**
+   * \brief Retrieves the configuration instances for the present (RobotWare) options.
+   *
+   * \return std::vector<cfg::sys::PresentOption> containing a list of the present (RobotWare) options.
+   */
+  std::vector<cfg::sys::PresentOption> getCFGPresentOptions();
+
   /**
    * \brief A method for retrieving the RobotWare options present on the active robot controller system.
    *
-   * \return std::vector<OptionInfo> containing a list of the present RobotWare options.
+   * \return std::vector<RobotWareOptionInfo> containing a list of the present RobotWare options.
    */
-  std::vector<RobotWareOptionInfo> getPresentRobotWareOptions();
+  ABB_LIBRWS_DEPRECATED std::vector<RobotWareOptionInfo> getPresentRobotWareOptions();
```

Comment:
Might be nice to tell users which function they should use as a replacement if this is deprecated.

Reply:
I added a comment about the replacement function in 57aa977